demo/src/main/java/com/jira/demo/model/Squad.java

```diff
@@ -30,8 +33,8 @@ public class Squad {
     @Column
     private String squadLeader;
 
+    // not sure about all
     @OneToMany(mappedBy = "squad", cascade = CascadeType.ALL)
```

Comment:
Same here: not sure about cascade type though, if you `delete` (or let say dismiss) a squad, all employees will be deleted (or fired). We don't want to have that situation